### PR TITLE
LPD-17882 Fix failing test

### PR DESCRIPTION
**Pull request by adolfopa**
# What is this trying to solve?

https://liferay.atlassian.net/browse/LPD-17882

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

File: modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java [hunk 1]
```diff
@@ -1333,9 +1332,8 @@ public List<MBMessage> getGroupUserMessageBoardMessagesActivity(
 			MBMessageTable.INSTANCE
 		).where(
 			MBMessageTable.INSTANCE.modifiedDate.in(
-				DSLQueryFactoryUtil.select(
-					DSLFunctionFactoryUtil.max(
-						MBMessageTable.INSTANCE.modifiedDate)
+				DSLQueryFactoryUtil.selectDistinct(
+					MBMessageTable.INSTANCE.modifiedDate
```
Comment (marcogalluzzi) [inline modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java:1336]: I haven't go trough very deeply, but why in one case you use the `selectDistinct` and in the other just `select`?

Comment (adolfopa) [reply modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java:1336]: You're right. I'll push-force the change..

**Comment by marcogalluzzi** [comment]
ci:forward

**Pull request closed without merging**